src/components/pages/complete-password-reset.yml

```diff
@@ -22,3 +22,9 @@ passwordChangedSuccessfully:
 redirectMessage:
   en: You will be redirected to the <0>logon page</0> in {seconds, plural, =1 {# second} other {# seconds}}.
   de: Sie werden in {seconds, plural, =1 {# Sekunde} other {# Sekunden}} auf die <0>Anmeldeseite</0> weitergeleitet.
+passwordChangeFailure:
+  en: The password changing code is invalid or expired.
```

Comment:
I guess "The password reset code is invalid..." would have been better here, we keep talking about password reset (not changing) :).

Reply:
Was already merged when you left the comment, but I can adjust directly on master, thanks.